R/shift_configuration.R

```diff
@@ -27,7 +27,8 @@
 #' \item{Y}{input trait vector/matrix.}
 #' \item{tree}{input tree.}
 #' \item{shift.configuration}{estimated shift positions, i.e. vector of indices of edges where the estimated shifts occur.}
-#' \item{shift.values}{estimates of the shift values.}
```

Comment:
it's in the code too: line 853. Sabrina: no need to make the change if Mohammad does it. You'll have to pull his changes anyway, or else you'll have to merge his change with yours.